SU-SWS/stanford_basic
# D8CORE-930: Accessible Skip Link Anchor

# READY FOR REVIEW 

# Summary
- Replaces the anchor jump link target from a div with an ID to a named anchor as requested by level access. 

# Review By (Date)
- Wednesday Jan 29th.

# Urgency
- Low

# Steps to Test

1. Build a site locally
2. Check out this branch
3. Clear caches
4. Validate skiplink jumps to desired location and no additional visual changes are apparent with the adding of the link tag.

# Affected Projects or Products
- D8CORE
- Accessibility

# Associated Issues and/or People
- D8CORE-930

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
